data/remnant/remnant.txt

```diff
@@ -475,7 +500,109 @@ phrase "friendly remnant"
 		"."
 		"!"
 
-phrase "friendly remnant"
+phrase "remnant no scan"
+	word
+		"Please"
+	word 
+		" "
+	word
+		"do not"
+		"try not to"
+	word
+		" "
+	word
+		"scan"
+	word
+		" "
+	word
+		"us."
+		"our ships."
+
+phrase "remnant no scan"
+	word
+		"Careless scans costs lives."
+		"Unasked for scans are unwelcome."
+		"Scanning is a privilege to be earned."
+
+phrase "remnant scan"
+	word
+		"Please"
```

Comment:
This does not work with the rest of this phrase.

Reply:
Thanks. This will be replaced in the next push.